app/src/main/res/values/strings.xml

```diff
@@ -186,6 +186,10 @@
     <string name="space_trackpad">Space bar trackpad</string>
     <!-- Description for "space_trackpad" option. -->
     <string name="space_trackpad_summary">Swipe on the spacebar to move the cursor</string>
+    <!-- Preferences item for enabling inserting more spaces key -->
+    <string name="insert_more_spaces">Insert spaces after punctuation</string>
+    <!-- Description for "insert_more_spaces" option. -->
+    <string name="insert_more_spaces_summary">Automatically insert space after punctuation, even if the previous word was not a suggestion</string>
```

Comment:
Nice, everything is perfect.
Can you change title to "Autospace after punctuation" and replace ", even if the previous word was not a suggestion" by  "when typing a new word." ? This will be more clear for a regular user. Thanks

Reply:
And maybe change string ids to more explicit ones such as "autospace_after_punctuation" and "autospace_after_punctuation_summary".

Reply:
Will do!

So the issues in https://github.com/openboard-team/openboard/pull/564#issuecomment-1069557899 will be tackled a new PR.

Reply:
Thanks! 
We can create an issue to remember this, I'll try to have a closer look when I can.